endpoints/openrtb2/amp_auction.go

```diff
@@ -352,6 +368,8 @@ func sendAmpResponse(
 						glog.Errorf("/openrtb2/amp Critical error unpacking targets: %v", err)
 						ao.Errors = append(ao.Errors, fmt.Errorf("Critical error while unpacking AMP targets: %v", err))
 						ao.Status = http.StatusInternalServerError
+						seatNonBid.Append(getNonBidsFromStageOutcomes(hookExecutor.GetOutcomes()))
```

Comment:
Related to my above comment, `seatNonBid` could be `nil` here.. leading to a confusing situation.

Reply:
Agree, made correction. Now passing the seatNonBid as a copy.